pkg/operator/controller/garden/care/health_test.go

```diff
@@ -64,8 +62,6 @@ var _ = Describe("Garden health", func() {
 	)
 
 	BeforeEach(func() {
-		DeferCleanup(test.WithFeatureGate(features.DefaultFeatureGate, features.HVPA, true))
```

Comment:
Some integration tests have to be adapted as well:
- https://github.com/gardener/gardener/blob/f3efe83358ef0a6182d63c759b290bd392d7260f/test/integration/gardenlet/seed/seed/seed_test.go#L193
- https://github.com/gardener/gardener/blob/f3efe83358ef0a6182d63c759b290bd392d7260f/test/integration/operator/garden/garden/garden_test.go#L81